src/dataExplorer/components/resources/types/script/index.ts

```diff
@@ -16,8 +16,8 @@ const {getScript, patchScript, postScript} = CLOUD
       postScript: false,
     }
 
-export default function script(register) {
-  register({
+export default () => {
```

Comment:
```suggestion
export const scriptResourceRegistration = () => {
```

Suggest to use named exports to help with https://github.com/influxdata/ui/issues/6007